### ISSUE TITLE
add `helm repo update`

### ISSUE THREAD

**Issue by ipedrazas** [ipedrazas/drone-helm#20]
Right now, there's no possibility in updating the repo

**Comment by so0k** [comment]
I don't think this is needed anymore, `helm repo add` seems to download the latest repo index anyway - so every run starts with an updated repository?

**Comment by danielkrainas** [comment]
This is the default behavior for helm now and can probably be closed, see kubernetes/helm#1142

From the PR's author:
> Just as a reminder: this PR changes the default behaviour for helm repo add so that it updates existing repository entries. I've also added a --no-update flag for those who prefer the old behaviour.
